.editorconfig

```diff
@@ -1,5 +1,8 @@
+root = true
+
 # All files
 [*]
+max_line_length = 80
```

Comment:
this should be consistent with the max-line-length in .flake8

Reply:
added it to python